patches/@types+ejson+2.1.3.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@types/ejson/index.d.ts b/node_modules/@types/ejson/index.d.ts
+index 3a35636..278ef98 100755
+--- a/node_modules/@types/ejson/index.d.ts
++++ b/node_modules/@types/ejson/index.d.ts
+@@ -17,7 +17,7 @@ export function parse(str: string): any;
+ export function stringify(obj: any, options?: StringifyOptions): string;
+ 
+ export function toJSONValue(obj: any): string;
+-export function fromJSONValue(obj: string): any;
++export function fromJSONValue(obj: Object): any;
```

Comment:
I opened an PR with these changes:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58819

But for today we need a patch :(